server/remark-lint-messaging.ts

```diff
@@ -81,12 +81,22 @@ function checkMessaging(
     const re = new RegExp(rule.incorrect);
     const badText = text.match(re);
     if (text.match(re)) {
+      let extras = "";
+      if (part == PageLocation.Body || part == PageLocation.Headers) {
+        extras +=
+          " Note that the line number is not accurate if the issue occurs inside a partial.";
```

Comment:
If we are adding this _after- text, is a space sufficient? Do we need to add any extra punctuation or parentheses or something? Looks good to me regardless, just chattin